### PR TITLE
Phase 2 sweeps

### PR DESCRIPTION
**Pull request by gabrieltseng**
This implements the following $lr \times wd$ sweeps for our four model sizes (nano, tiny, base, large):

| size | learning rates | weight decays |
| --- | --- | --- |
| nano | `0.001, 0.002, 0.003` | `0.01, 0.02, 0.03`|
| tiny | `0.001, 0.002, 0.003` | `0.01, 0.02, 0.03`|
| base | `0.0001, 0.0002, 0.0003` | `0.01, 0.02, 0.03`|
| large | `0.00005, 0.0001, 0.0002` | `0.01, 0.02, 0.03`|

I have sort of tested this works ([link](https://beaker.allen.ai/orgs/ai2/workspaces/earth-systems/work/01K6N4922VYTSV871DV7RXBHHT?taskId=01K6N492383AW79GYJ2B8CGDX6&jobId=01K6N4926P1VFVD3CQKBDA6N8C))? But I expect there will be lots of tweaks necessary as we actually kick them off. 

Some final things on my mind:
- do we want a smaller MLP ratio following Yawen's experiments? This could do nice things for model size

### CODE REVIEW

File: scripts/2025_10_02_phase2/large_launch.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
...
+python scripts/2025_10_02_phase2/base.py launch base_lr0.0001_wd0.03 ai2/titan  --train_module.optim_config.lr=0.0001 --train_module.optim_config.weight_decay=0.03
+python scripts/2025_10_02_phase2/base.py launch base_lr0.0002_wd0.01 ai2/titan  --train_module.optim_config.lr=0.0002 --train_module.optim_config.weight_decay=0.01
+python scripts/2025_10_02_phase2/base.py launch base_lr0.0002_wd0.02 ai2/titan  --train_module.optim_config.lr=0.0002 --train_module.optim_config.weight_decay=0.02
+python scripts/2025_10_02_phase2/base.py launch base_lr0.0002_wd0.03 ai2/titan  --train_module.optim_config.lr=0.0002 --train_module.optim_config.weight_decay=0.03
+python scripts/2025_10_02_phase2/base.py launch base_lr0.00005_wd0.01 ai2/titan  --train_module.optim_config.lr=0.00005 --train_module.optim_config.weight_decay=0.01
+python scripts/2025_10_02_phase2/base.py launch base_lr0.00005_wd0.02 ai2/titan  --train_module.optim_config.lr=0.00005 --train_module.optim_config.weight_decay=0.02
+python scripts/2025_10_02_phase2/base.py launch base_lr0.00005_wd0.03 ai2/titan  --train_module.optim_config.lr=0.00005 --train_module.optim_config.weight_decay=0.03
```
Comment (gabrieltseng) [inline scripts/2025_10_02_phase2/large_launch.sh]: Trying a smaller learning rate for large

File: scripts/2025_10_02_phase2/script.py [hunk 2]
```diff
@@ -0,0 +1,321 @@
+"""Trying to prototype fitting everything into olmo core."""
```
Comment (gabrieltseng) [inline scripts/2025_10_02_phase2/script.py:1]: We will need to update this script with the final recipe

File: scripts/2025_10_02_phase2/script.py [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+from olmo_core.train.checkpoint import CheckpointerConfig
+from olmo_core.train.common import Duration, LoadStrategy
+from olmo_core.train.config import TrainerConfig
+from upath import UPath
+
+from helios.data.concat import HeliosConcatDatasetConfig
+from helios.data.constants import Modality
+from helios.data.dataloader import HeliosDataLoaderConfig
+from helios.data.dataset import HeliosDatasetConfig
+from helios.internal.common import build_common_components
```
Comment (gabrieltseng) [inline scripts/2025_10_02_phase2/script.py:28]: The default is for the run priority to be high. Do we want it to be urgent?

File: scripts/2025_10_02_phase2/tiny_launch.sh [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+python scripts/2025_10_02_phase2/tiny.py launch tiny_lr0.002_wd0.03 ai2/ceres-cirrascale  --train_module.optim_config.lr=0.002 --train_module.optim_config.weight_decay=0.03 --launch.clusters='[ai2/ceres-cirrascale,ai2/jupiter-cirrascale-2]'
+python scripts/2025_10_02_phase2/tiny.py launch tiny_lr0.003_wd0.01 ai2/ceres-cirrascale  --train_module.optim_config.lr=0.003 --train_module.optim_config.weight_decay=0.01 --launch.clusters='[ai2/ceres-cirrascale,ai2/jupiter-cirrascale-2]'
+python scripts/2025_10_02_phase2/tiny.py launch tiny_lr0.003_wd0.02 ai2/ceres-cirrascale  --train_module.optim_config.lr=0.003 --train_module.optim_config.weight_decay=0.02 --launch.clusters='[ai2/ceres-cirrascale,ai2/jupiter-cirrascale-2]'
+python scripts/2025_10_02_phase2/tiny.py launch tiny_lr0.003_wd0.03 ai2/ceres-cirrascale  --train_module.optim_config.lr=0.003 --train_module.optim_config.weight_decay=0.03 --launch.clusters='[ai2/ceres-cirrascale,ai2/jupiter-cirrascale-2]'
```
[the remark this answers is not in the document]
Comment (gabrieltseng) [reply scripts/2025_10_02_phase2/tiny_launch.sh]: not a mistake

File: scripts/2025_10_02_phase2/large_launch.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (gabrieltseng) [reply scripts/2025_10_02_phase2/large_launch.sh]: thanks cursor!

File: scripts/2025_10_02_phase2/nano_launch.sh [hunk 5]
```diff
@@ -0,0 +1,9 @@
...
+python scripts/2025_10_02_phase2/nano.py launch nano_lr0.002_wd0.03 ai2/ceres-cirrascale  --train_module.optim_config.lr=0.002 --train_module.optim_config.weight_decay=0.03 --launch.clusters='[ai2/ceres-cirrascale,ai2/jupiter-cirrascale-2]'
+python scripts/2025_10_02_phase2/nano.py launch nano_lr0.003_wd0.01 ai2/ceres-cirrascale  --train_module.optim_config.lr=0.003 --train_module.optim_config.weight_decay=0.01 --launch.clusters='[ai2/ceres-cirrascale,ai2/jupiter-cirrascale-2]'
+python scripts/2025_10_02_phase2/nano.py launch nano_lr0.003_wd0.02 ai2/ceres-cirrascale  --train_module.optim_config.lr=0.003 --train_module.optim_config.weight_decay=0.02 --launch.clusters='[ai2/ceres-cirrascale,ai2/jupiter-cirrascale-2]'
+python scripts/2025_10_02_phase2/nano.py launch nano_lr0.003_wd0.03 ai2/ceres-cirrascale  --train_module.optim_config.lr=0.003 --train_module.optim_config.weight_decay=0.03 --launch.clusters='[ai2/ceres-cirrascale,ai2/jupiter-cirrascale-2]'
```
[the remark this answers is not in the document]
Comment (gabrieltseng) [reply scripts/2025_10_02_phase2/nano_launch.sh]: not a mistake - fixed the PR description
